bin/build.ps1

```diff
@@ -34,17 +34,12 @@ if (-Not $skiptemplates) {
 <# Cleanup of old files #> 
 Write-Host "Cleaning before new build" -ForegroundColor Green
 & stack clean
-if (test-path "$deckerdir\resource\decker\support\vendor") {
-    Remove-Item "$deckerdir\resource\decker\support\vendor" -Recurse -Force -ErrorAction Continue
```

Comment:
What does this have to do with "accesible handout mode"?

Reply:
Nothing, I just happened to mention this while working inside the branch and accidentally added it here - we should cherry pick that out into develop and master because it deletes the vendor directory that has been statically added to the support directory.